changelogs/fragments/943-enhance-Add-wait-zos-operator-and-query.yml

```diff
@@ -0,0 +1,6 @@
+enhancements:
```

Comment:
enhancements is not a valid tag for changelogs, rather use minor_changes or major_changes. I think minor_changes will be ok.

Reply:
changed